raysort/main_push.py

```diff
@@ -296,6 +296,9 @@ def _submit_map(map_id: int) -> ray.ObjectRef:
     ray.get(all_map_out)
     logging.info("All map tasks finished; start reduce stage")
 
+    if cfg.skip_final_reduce:
```

Comment:
Actually, this is not enough to make sure all shuffle finishes. In `MergeController.reduce()`, `_finish_merge_results()` makes sure all merge tasks are done. So let's return [] there instead.